src/components/Datasets/index.js

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable no-console */
```

Comment:
1. Why no console?
2. What's the point of this component?

Reply:
@kilemensi Sorry, removed the console comment. This component is to showcase how the Dataset component.